api.bs

```diff
@@ -1110,6 +1114,48 @@ if the user has opted out of collection of diagnostic data.
 * User ability to view the impression store and past report submissions.
 
 
+# HTTP API # {#http-api}
+
+\`<dfn http-header><code>Save-Impression</code></dfn>\` is a
+[=structured header/dictionary|Dictionary Structured Header=]
+set on a response requesting that the user agent invoke the
+<a method for=PrivateAttribution>saveImpression()</a> API.
+
+<pre class=example id=ex-save-impression-header>
+Save-Impression: conversion-site=advertiser.example, histogram-index=2, filter-data=12, lifetime-days=7
```

Comment:
```suggestion
Save-Impression: conversion-site="advertiser.example", histogram-index=2, filter-data=12, lifetime-days=7
```

In the spirit of making progress, I think we should follow the advice in RFC 9651 and stick with strings.  Tokens can represent any domain name, so we *could* include tokens or even switch to tokens, but it makes sense to follow the well-lit path on this one.